src/stores/GuildEmojiRoleStore.js

```diff
@@ -109,4 +120,6 @@ class GuildEmojiRoleStore extends DataStore {
    */
 }
 
+Util.mixin(GuildEmojiRoleStore, ['set']);
```

Comment:
Is a mixin here really necessary?
What problem did it cause / couldn't be solved by regular inheritance?

Reply:
The issue is that there is nothing to extend because the class itself is empty, so I need to intercept what happens when methods are called to apply to `_filtered`, as the data is there as a getter.